cranelift-codegen/src/scoped_hash_map.rs

```diff
@@ -4,10 +4,12 @@
 //! container that has a concept of scopes that can be entered and exited, such that
 //! values inserted while inside a scope aren't visible outside the scope.
 
-use crate::fx::FxHashMap;
+use crate::fx::{FxHashMap, FxHasher};
 use core::hash::Hash;
 use core::mem;
 
+type Hasher = core::hash::BuildHasherDefault<FxHasher>;
```

Comment:
Could this be guarded by the not(feature = std) too, please? I think it's unused otherwise, and causes [compiler warnings](https://dev.azure.com/CraneStation/Cranelift/_build/results?buildId=988&view=logs&jobId=e0fca624-2401-5624-f540-1c188feac080&taskId=90593842-716e-5c1d-6fac-3c5c8d07c009&lineStart=197&lineEnd=198&colStart=1&colEnd=1).